src/neuroconv/datainterfaces/ecephys/alphaomega/alphaomegadatainterface.py

```diff
@@ -1,5 +1,6 @@
+from pydantic import DirectoryPath
```

Comment:
I tested this out here as a showcase of how the rest of the interfaces would be refactored, but left that for a followup since it's a lot of changes all at once

There will technically be a small back-compatibility break until that follow-up is merged, in the 'format' of the path type in each schema. However, this field is not used by JSON validation at all so it's not expected to break anything in the interim